Feature/databricks changes

agnessnowplow opened the pull request:
Adding the Databricks support in a new branch as there were too many merge conflicts using the previous branch that was built before the major styling changes happened.

Not sure if the UI is easy enough to navigate having different warehouses at the moment. (perhaps a decision matrix of some sort could be added with all the options for each chapter?)

I also left some currently superfluous filters within the notebook as it does not seem to work on dates but perhaps there is a way to solve this.

Jack-Keene commented on content/_index.md:
```diff
@@ -65,14 +65,14 @@ gantt
   - New dbt project created and configured
 - Python 3 Installed
-- Snowflake account and a user with access to create schemas and tables
+- Snowflake / Databricks account and a user with access to create schemas and tables
```
Could we change this to Snowflake or Databricks account? It might look like they need both for this

agnessnowplow replied:
I also changed line 65 as it is the same situation there. For me the / means optional or choice but for sure let's change them if it causes ambiguity.

Jack-Keene commented on content/next_steps/next_steps_2.md:
````diff
@@ -21,3 +25,14 @@ Run the command below to run the streamlit locally:
 ```
+{{% /tab %}}
+
+{{% tab name="Databricks Notebook" %}}
````
Could we name the tabs just 'Streamlit' and 'Databricks'?

Jack-Keene reviewed (commented):
Looks good, just a few comments on formatting. I also think it would also be good to have some screenshots of the Databricks notebook for the introduction.

agnessnowplow commented:
Thanks Jack, I have made the changes. I also rearranged the databricks dashboard chart, removed the unused filter widget and changed the ipython notebook to a zip file to be able to actually download it.

paulboocock commented on content/upload/upload_2.md:
```diff
@@ -0,0 +1,200 @@
+{{% /attachments %}}
+
+
+You can load the sample data to the warehouse using your `Databricks Workspace` as described in the below steps. Please download the *sample_events.csv* from the attachments. For more details please check out the official [Databricks documentation](https://docs.databricks.com/ingestion/add-data/index.html).
```
This might be out of scope for this release, but you can do Delta Sharing within Databricks now - @georgewoodhead did this for the composable CDP accelerator with Databricks. It means the assets are also available for others to use, no imports required! I wonder if we can do the same here, don't block the release for this, but have a chat to George when he's back to see if we could do that here too

agnessnowplow replied:
Thanks, Paul, I'll talk to him about this!
